strawberry/custom_scalar.py

```diff
@@ -0,0 +1,56 @@
+from graphql.type.scalars import GraphQLScalarType
+
+from .type_converter import REGISTRY
+
+
+def _process_scalar(cls, *, description, serialize, parse_value, parse_literal):
+    name = cls.__name__
+
+    REGISTRY[cls] = GraphQLScalarType(
+        name=name,
+        description=description,
+        serialize=serialize,
+        parse_value=parse_value,
+        parse_literal=parse_literal,
+    )
+
+    return cls
+
+
+def scalar(cls=None, *, description=None, serialize, parse_value, parse_literal=None):
```

Comment:
what if we add a default serializer and parse value? This could make it easy to implement scalars that are just aliases of normal types, like:

```
Currency = strawberry.scalar(int, description="Currency in cents.")
Code = strawberry.scalar(str, description="Conference code.")
```

Reply:
Note that the could would need to actually look like this (if you don't want to overwrite the handling of all ints and strings):
```python
Currency = strawberry.scalar(NewType("Currency", int), description="Currency in cents.")
Code = strawberry.scalar(NewType("Code", str), description="Conference code.")
```

I added a commit implementing this.